packages/dashboard/src/components/Recommendation/RecommendationBanner.base.tsx

```diff
@@ -0,0 +1,246 @@
+                {recommendationButtonLocalizedName}
+              </PrimaryButton>
+            </div>
+          </div>
+          <div className={this.classNames.recommendationContentRowVisualization}>
+            {recommendationVisualization ? this._getVisualizationComponent() : this.props.children}
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  private _generateFlexRowLayout(): JSX.Element {
+    const {
+      recommendationDescriptionHeader,
+      recommendationDescription,
+      recommendationButtonLocalizedName,
+      recommendationButtonAriaDescription,
+      handleViewRecommendationClick,
+      recommendationVisualization
+    } = this.props;
+
+    return (
+      <div className={this.classNames.recommendationContainer}>
+        <div className={this.classNames.recommendationTextContainer}>
+          <div className={this.classNames.recommendationHeader}>
+            <AutoFontSize
+              text={recommendationDescriptionHeader}
+              targetElementType={'div'}
+              fontSizeMapping={[{ fontSize: 28, lineHeight: '36px' }, { fontSize: 16, lineHeight: '23px' }]}
+              targetLines={2}
+            />
+          </div>
+          <div className={this.classNames.recommendationContent}>{recommendationDescription} </div>
+          <div>
+            <PrimaryButton
+              data-automation-id="btnRecommendation"
+              name={recommendationButtonLocalizedName}
+              onClick={handleViewRecommendationClick}
+              ariaDescription={recommendationButtonAriaDescription}
+            >
+              {recommendationButtonLocalizedName}
+            </PrimaryButton>
+          </div>
+        </div>
+        <div className={this.classNames.recommendationVisualizationContainer}>
+          {recommendationVisualization ? this._getVisualizationComponent() : this.props.children}
+        </div>
+      </div>
+    );
+  }
+
+  private _getVisualizationComponent(): JSX.Element | null {
+    const { recommendationVisualization } = this.props;
+    switch (recommendationVisualization) {
+      case VisualizationType.ImageIllustration:
+        return this._renderImageIllustrationVisualization();
+      case VisualizationType.MultiStackBarChart:
+        return this._renderStackedBarChartVisualization();
+      default:
+        return null;
+    }
+  }
+
+  private _renderImageIllustrationVisualization(): JSX.Element | null {
+    const { imageVisualizationSrc } = this.props;
+    if (imageVisualizationSrc) {
+      return (
+        <div className={this.classNames.imageIllustrationContainerStyle}>
+          <img src={imageVisualizationSrc} className={this.classNames.imageIllustrationStyle} />
+        </div>
+      );
+    }
+
+    return null;
+  }
+
+  private _renderStackedBarChartVisualization(): JSX.Element | null {
+    const { chartVisualizationData } = this.props;
+    const legendColors = ['#0078D4', '#0B6A0B', '#662D91', '#038387', '#00AE56'];
```

Comment:
>0078D4 [](start = 28, length = 6)

Is this something we can get from the color palate?

Reply:
I have a pending follow up with PMs on this. The current design requires us to use these colors, will do a separate PR to handle this once we have the appropriate palette colors.

---
In reply to: [237587139](https://github.com/OfficeDev/office-ui-fabric-react/pull/7249#discussion_r237587139) [](ancestors = 237587139)